src/js/fiContainer.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var divisionHeader = require('./divisionHeader.jsx');
+var fiStatus = require('./fiStatus.jsx');
+
+var fiContainer = React.createClass({
+
+  getInitialState: function(){
+    return this.updateDivisions(this.props.institutions);
+  },
+
+  updateDivisions: function(institutions){
```

Comment:
Can we use a different name other than `updateDivisions`?

Seems like this is grouping each institution by the status, `updateInstitutionsByStatus` maybe? Or even `sortInstitutionsByStatus`?

If so, I think the `state.divisions` array below (~line 13) could be changed, and the `divisionHeader.jsx` filename and vars within.